src/pages/UpdateContactPage.tsx

```diff
@@ -84,13 +84,13 @@ const UpdateContactPage: FC = () => {
                         {error && <ErrorBlock text={error}/>}
                         <ContactForm
                             initValues={{
-                                email: contact.email,
-                                firstName: contact.firstname,
-                                lastName: contact.lastname,
-                                jobTitle: contact.jobtitle,
-                                phone: contact.phone,
-                                ownerId: contact.hubspot_owner_id,
-                                lifecycleStage: contact.lifecyclestage
+                                email: contact?.email || "",
```

Comment:
<img width="599" alt="image" src="https://github.com/user-attachments/assets/4bd56038-9972-4350-ad5b-ff9cd735232a">
<img width="591" alt="image" src="https://github.com/user-attachments/assets/7561f2fc-8b34-4e1a-8dfb-1532de32642b">

It would be nice to get typescript to see the objects we are dealing with here. If I change it to `contact: contact?.data?.properties ?? null,` then I get the below typing which correctly says `{...}|null`. Can `lodash` be removed in that function so the typings are accurate please?
<img width="594" alt="image" src="https://github.com/user-attachments/assets/2e7efc30-af08-4db7-8c99-2c527c4df326">

Reply:
I haven't changed anything and I have the correct types
<img width="591" alt="Screenshot 2024-07-25 at 17 57 26" src="https://github.com/user-attachments/assets/dde3fcb6-8b3f-4e86-83a7-8c0f9faa9c7f">